ppa/archive/solr.py

```diff
@@ -91,6 +91,27 @@ class SolrSchema(object):
                 ]
             },
         },
+        {
+            'name': 'punctuation_parser',
+            "class":"solr.TextField",
+            # for now, configuring index and query analyzers the same
+            # if we want synonyms, query must be separate
+            "analyzer": {
+                # "charFilters": [],
+                "tokenizer": {
+                    "class": "solr.WhitespaceTokenizerFactory",
+                },
+                "filters": [
+                    {"class": "solr.StopFilterFactory", "ignoreCase": True,
```

Comment:
I think we should omit the stop word filter factory - I'm hoping this will let us search on A/B rhyme scheme notation, so we don't want to throw out all the "a"s.